### ISSUE TITLE
Unable to see any exception or logging message after importing mediapipe

### ISSUE THREAD

**Issue by Ziwen510** [google/mediapipe#4552]
### Have I written custom code (as opposed to using a stock example script provided in MediaPipe)

None

### OS Platform and Distribution

Windows 11

### Programming Language and version

Python 3.9.12

### Solution

any

### Describe the actual behavior

```shell
Finish without any output or exception.
```

### Describe the expected behaviour

```shell
show the raised ValueError
```

### Standalone code/steps you may have used to try to get what you need

```shell
# Create a Python file anywhere and type the following

import mediapipe

raise ValueError("12345")
```

### Other info / Complete Logs

```shell
I can get the ValueError once I comment the import.
It happens both in my cmd terminal and vscode terminal.
I use the latest 0.10.1.
```

**Comment by kuaashish** [comment]
@Ziwen510,

Could you please elaborate your query with complete details. Thank you!!

**Comment by Ziwen510** [comment]
I do want to elaborate more but the problem is so simple. I record the
screen for your reference.
For example, I can reproduce it by just creating a Python file in my root
directory.
 screen recording.mp4
<https://drive.google.com/file/d/1FYjuKSk_dIOcn7s6owGNoDXhRuGWoxbQ/view?usp=drive_web>

It's not limited to this certain Error but the same for any exceptions or
logging messages.
Thank you for your help!

**Comment by kuaashish** [comment]
@Ziwen510,

It seems shared drive link is not accessible. Please share the link which can be accessible to understand the issue better.

**Comment by Ziwen510** [comment]
Sorry. The drive link should be accessible now. Thank you!

**Comment by Ziwen510** [comment]
It's a serious problem. And it seems that others are experiencing the same problem (see the last comment in this issue): https://github.com/google/mediapipe/issues/4474#issuecomment-1609242770
Looking forward to your reply!

**Comment by kuaashish** [comment]
Hello @schmidt-sebastian,

Could you please look into this issue. Thank you

**Comment by Muneeb-Ahmed-Khan** [comment]
I have upgraded to mediapipe 0.10.2 but I am still facing the same issue. Please resolve it as soon as possible.

**Comment by kuaashish** [comment]
Hi @Ziwen510,

The issue has been addressed in the latest MediaPipe version [0.10.7](https://github.com/google/mediapipe/releases/tag/v0.10.). I was able to successfully reproduce the exception message using the steps you provided. The validation was conducted with **"Python Version 3.10.11"** on **"Windows 11 Home Single Language Version 22H2"**, and the corresponding screenshot is attached below:

<img width="851" alt="ValueError" src="https://github.com/google/mediapipe/assets/98159216/e8f42f4b-515f-419b-9b24-2687c268fc5f">



Please proceed with the build using the aforementioned configurations, and inform us promptly if any issues persist.


Thank you

**Comment by kuaashish** [comment]
Hi @Ziwen510,

Could you please provide an update on the current status of the reported issue? If the issue has been resolved from your end, we can proceed to mark it as resolved and close it.

Thank you
